### DISCUSSION TITLE
Using Traffic Shapers Independently - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#803] [Showcases]
Discuss the "Using Traffic Shapers Independently" showcase here. Ideas and recommendations are welcome.

**Comment by Lukas327** [comment]
In the last paragraph should be "max residence time/average production interval=10ms/0.5ms=20" instead of " average production interval / max residence time = 10ms / 0.5ms = 20"

**Reply by adamgeorge309** [reply]
You are correct, thanks for pointing this out. Fixed in master.
